expyfun/stimuli/_hrtf.py

```diff
@@ -49,20 +62,56 @@ def _get_hrtf(angle, source, fs):
     if angle < 0:
         leftward = True
         read_angle = -angle
-    if read_angle not in angles:
+    if read_angle not in angles and not interp:
         raise ValueError('angle "{0}" must be one of +/-{1}'
                          ''.format(angle, list(angles)))
     brir = data['brir']
-    idx = np.where(angles == read_angle)[0]
-    assert len(idx) == 1
-    brir = brir[idx[0]].copy()
+    if not interp:
+        idx = np.where(angles == read_angle)[0]
+        assert len(idx) == 1
+        brir = brir[idx[0]].copy()
+    elif interp:
+        idx = np.where(angles < read_angle)[0]
+        # angles are
+        b = angles[idx[-1]]
+        c = angles[idx[-1] + 1]
```

Comment:
Hmmm, you're right. I think this goes back to the fact that there's no need for interpolation if the angle's hrtf is known.